client/go/outline/electron/client_others.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2025 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !windows
+
+package main
+
+import "github.com/Jigsaw-Code/outline-apps/client/go/outline"
+
+func newOutlineClient(transportConfig string, adapterIp string, adapterIndex int) (*outline.Client, error) {
```

Comment:
Return unsupported instead. See may comment on main

Reply:
I'd prefer panic, since it's a must-fix development bug.

Reply:
panic is hard to track though. You will be failing in the caller already, so the developer will know no matter what.

Reply:
Updated.